app/src/main/java/org/dhis2/usescases/teiDashboard/dashboardfragments/teidata/TEIDataFragment.kt

```diff
@@ -438,12 +438,8 @@ class TEIDataFragment : FragmentGlobalAbstract(), TEIDataContracts.View {
         val model = dashboardViewModel.dashboardModel.value
         if (model is DashboardEnrollmentModel) {
             SchedulingDialog.newSchedule(
-                enrollment = model.currentEnrollment,
-                programStages = if (programStage != null) {
-                    listOf(programStage)
```

Comment:
programStage parameter is not needed any more?

Reply:
Yep. We are loading this information in the view model and setting it.